backend/src/api/futureInterview.js

```diff
@@ -93,35 +45,182 @@ router.post(
   })
 )
 
-router.get(
-  '/',
-  [leadsOnly],
+router.post(
+  '/uploadCandidates',
   errorWrap(async (req, res) => {
-    const futureInterviews = await FutureInterview.find()
-    let interviews = []
-    let code = 500
-    // TODO: do stuff
-    // retrieve list of FutureInterviews and return them
+    var workbook = XLSX.read(req.body['data'], { type: 'binary' })
```

Comment:
Prefer `let` over `var` for [tighter scoping](https://www.geeksforgeeks.org/difference-between-var-and-let-in-javascript/)

Reply:
But I also don't see these modified after creation - is there a reason why we're not using `const`?

Reply:
Yep - that's definitely a typo. Good catch.